ckanext/cfpb_extrafields/exportutils.py

```diff
@@ -0,0 +1,108 @@
+"""Export all datasets to excel-compatible csv.
+
+Currently only exports public datasets, but once CKAN is upgraded to v2.4+,
+the code can be updated to support private datasets.
+"""
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from stringIO import StringIO
+import csv
+import json
+
+
+FIELDS = [
+    ("title", "Dataset Title"),
+    ("private", "Visibility"),
+    ("notes", "Description"),
+    # ("i", "Subject Matter"),
```

Comment:
Commented out fields are the ones that I couldn't find a suitable mapping for. I'll move them out of this data structure and into a comment above or below it.